checks.d/http_check.py

```diff
@@ -295,27 +313,21 @@ def _check(self, instance):
             if content_match:
                 # r.text is the response content decoded by `requests`, of type `unicode`
                 content = r.text if type(content_match) is unicode else r.content
-                if re.search(content_match, content, re.UNICODE):
-                    self.log.debug("%s is found in return content" % content_match)
-                    service_checks.append((
-                        self.SC_STATUS, Status.UP, "UP"
-                    ))
+                if not reverse_content_match and re.search(content_match, content, re.UNICODE):
```

Comment:
The code is correct but maybe something like this could be more readable:
```
if re.search(content_match, content, re.UNICODE):
    if reverse_content_match:
        # do smth
    else:
       # do smth else
else:
      if reverse_content_match:
          # do smth
      else:
          # do smth else
```
you could also move the `if reverse_content_match:` blocks within a separate function where you evaluate (and maybe send) the final status.